mongodb/src/test/scala/docs/scaladsl/MongoSinkSpec.scala

```diff
@@ -204,6 +217,29 @@ class MongoSinkSpec
 
       found mustBe empty
     }
+
+    "replace with replaceOne and codec support" in assertAllStagesStopped {
+      insertDomainObjectsRange()
+      val updatedObjects =
+        testRange.map(i => DomainObject(i, s"updated-first-property-$i", s"updated-second-property-$i"))
+
+      // #replace-one
+      val source = Source(testRange).map(
+        i =>
+          DocumentReplace[DomainObject](
+            filter = Filters.eq("_id", i),
```

Comment:
In case of `replaceOne` such filter is probably the only one filter that makes sense, so it would be nice to have some predefined filter which wraps this expression. It would be something like:
```
object FilterId {
  def apply[T](id: T): Bson = Filters.eq("_id", id)
}
```
What do you think?

Reply:
I would guess that's the usual use case as well, but the current `updateOne` or `replaceOne` APIs still expect a generic filter.  This API should be consistent.